Formula/caffe.rb

```diff
@@ -4,7 +4,7 @@ class Caffe < Formula
   url "https://github.com/BVLC/caffe/archive/1.0.tar.gz"
   sha256 "71d3c9eb8a183150f965a465824d01fe82826c22505f7aa314f700ace03fa77f"
   license "BSD-2-Clause"
-  revision 42
+  revision 43
```

Comment:
It really should be deprecated. The releasenotes for 1.0 mention that it's going into maintenance mode but it hasn't seen a commit in over 2 years and the successor has been abandoned in favour of pytorch

Reply:
Done in #119433. Let's rev bump this here though. We will have to rebase anyway as `opencv` did not build anyway.